pkg/controller/projectreference/projectreference_adapter.go

```diff
@@ -302,6 +346,15 @@ func (r *ReferenceAdapter) UpdateProjectID() error {
 	return r.kubeClient.Update(context.TODO(), r.ProjectReference)
 }
 
+func (r *ReferenceAdapter) UpdateServiceAccountName() error {
+	const serviceAccountNameSuffixLength = 8
+	// using k8s library instead of local implelemtation
```

Comment:
```suggestion
	// using k8s library instead of local implementation
```
also... this comment doesn't tell anything the code doesn't, probably not necessary